ios/Podfile

```diff
@@ -9,7 +9,7 @@ end
 
 target 'Plugin' do
   capacitor_pods
-  pod 'Firebase/Crashlytics', '8.12.1'
+  pod 'Firebase/Crashlytics', '>= 8.12.1'
```

Comment:
@robingenz I updated the podspec file.
I tested Crashlytics with 9.30 and works correctly.
If I refer to [CHANGELOG.md](https://github.com/firebase/firebase-ios-sdk/blob/master/Crashlytics/CHANGELOG.md) and after look for interface updates [here](https://github.com/firebase/firebase-ios-sdk/tree/master/Crashlytics/Crashlytics/Public/FirebaseCrashlytics), interface seems not updated so I consider the plugin works for other major versions.

Reply:
Maybe at the moment, but that does not mean that it will be the same in the future. You have to make sure that the plugin can only be used with compatible versions. 
May I ask why this fixed version is a problem at all?

Reply:
What do you mean about "fixed version" ?  
The version of library (v3.0.0) or the version of this PR ?  

However, related to the code of ios content, the library seems to be a wrapper of Ios Crashlytics SDK.
Suppose the interfaces of Crashlytics doesn't change, the wrapper will be compatible.
I think we can update the version constraint and if a change occurs, update the compatibility of library.
But maybe I'm missing something ?

Otherwise, I'm open to restreint the compatible version to 8.12.1 and 9.3.0 (for me) if you prefer but I don't know how configure it in podspec.

Reply:
> What do you mean about "fixed version" ?

The pod version (`8.12.1`) you want to change.
Why do you need a different version of the pod? What is this plugin missing?

>  Suppose the interfaces of Crashlytics doesn't change, the wrapper will be compatible.

That's what semantic versioning is for. Major versions can contain breaking changes.

Feel free to submit a PR which allows minor and patch updates, but no major updates.

Reply:
> Why do you need a different version of the pod? What is this plugin missing?

Because our app use Batch which require Firebase Core 9.30.0 and have breaking changes. After analyzing, this library which we use has no impact if we upgrade to 9.30.0.

> That's what semantic versioning is for. Major versions can contain breaking changes.

I know but for this wrapper, I didn't seen interface breaking changes for version 9.X and 10.X.

> Feel free to submit a PR which allows minor and patch updates, but no major updates.

What do you mean by this ? I don't update the versioning to let you associate the right version related to your versioning rule.

Reply:
> I know but for this wrapper, I didn't seen interface breaking changes for version 9.X and 10.X.

Yes, but that doesn't mean it will stay that way in the future.

> What do you mean by this ? I don't update the versioning to let you associate the right version related to your versioning rule.

There is a rule that allows minor and patch updates. That doesn't help you in this case but it would be a change I would accept as it still prevents breaking changes.

I have just published a blog post that shows how you can make these changes in your project: https://capawesome.io/blog/how-to-patch-a-capacitor-plugin/
This should solve your problem and this plugin can keep the current versioning.